src/audio/eq_iir/eq_iir.c

```diff
@@ -6,11 +6,11 @@
 //         Liam Girdwood <liam.r.girdwood@linux.intel.com>
 //         Keyon Jie <yang.jie@linux.intel.com>
 
+#include "eq_iir.h"
```

Comment:
Typically non-system include files come after system include files, so unless there's a reason to have eq_iir.h as first, I'd move this to L38 (or around there).